arch/x86/core/Kconfig.intel64

```diff
@@ -32,10 +32,24 @@ config EXCEPTION_STACK_SIZE
 	  support limited call-tree depth and must fit into the low core,
 	  so they are typically smaller than the ISR stacks.
 
+# We should really only have to provide one of the following two values,
+# but a bug in the Zephyr SDK for x86 precludes the use of division in
+# the assembler. For now, we require that these values be specified manually,
```

Comment:
Oh weird. We should probably file a GH ticket on this.

Reply:
Already done - the issue is in the Zephyr SDK repo, see https://github.com/zephyrproject-rtos/sdk-ng/issues/109 (to-do: figure out how to link issues across repos)